kubernetes/kfserving/README.md

```diff
@@ -269,9 +284,90 @@ The response is as below :
 }
 ```
 
+* For v2 protocol
```

Comment:
Will be better to split into 2 separate readmes for V1 and V2 protocol, with the main readme containing links for both